### DISCUSSION TITLE
Power spliter cable, network splitter cables

### DISCUSSION THREAD

**Discussion by quasigeek** [netbox-community/netbox#6698] [Q&A]
Hi, I have been searching but I cannot find an example.

Is there a was to define splitter cables? Like a Power splitter from C20 to 2xC13. Or a network cable that splits a 100Gb connection to 4x25Gb ( 100G QSFP28 to 4x25G SFP28 ), or an IB cable that splits 200Gb/s to 2x100Gb ( QSFP56 to 2xQSFP56 ).

All cable connections seems to be 1 to 1 only.

**Comment by candlerb** [comment]
To model Y power cables, I make a fake "device" with one power port and two power outlets, then connect three power cables.  That works fine because both power outlets receive power from the power port - and power is just power.

QSFP breakout is more difficult.  Netbox doesn't model either fibre strands or cable bundles (*).  It also doesn't directly model multiple interfaces on the same physical connector (although you can create 4 'virtual' interfaces and set their parent to be the port).  For now, your best bet is probably to model this as four interfaces, as if it were 4x25G SFP28, with four 1-1 cables going to the SFP28 endpoints - that's the only way to maintain the interface adjacencies.  If you model it as a 5-port breakout device then you'll lose the ability to trace through it.

(*) with the exception of rearport-to-rearport connections, which can carry multiple "positions", but there's no equivalent concept of  multiple positions on an interface

**Comment by jhofmueller** [comment]
I know this is more than three years old but AFAICS the topic is still open. Modeling a Y power cable as a device is not good enough because it 'breaks' the cable trace. I would love to have a trace from a PSU in a switch/server to the outlet it is plugged in. Any chance for a solution that satisfies this wish?
